docs/developer/decision-records/2023-03-15_repository_split/README.md

```diff
@@ -0,0 +1,77 @@
+# Splitting the Connector repository
+
+## Decision
+
+The Connector repository will be split up into a "core" repository and technology repositories. Extensions that are
+specific to a particular cloud environment will get moved out into new repositories to which we will refer as
+"technology repos".
+
+The Connector repository will _not_ get renamed, but it can henceforth be referred to as "core" or "connector-core".
+
+## Rationale
+
+Many third-party technology extensions are very specific to a particular cloud provider, and cannot reasonably be used
+across multiple clouds, thus they depend on specific libraries/SDKs and testing them may even require an account with
+that cloud provider.
+
+The reasoning for moving these kinds of extensions out to separate repositories is as follows:
+
+- keeping the core small and agile: building and running it should be fast and efficient, no third-party account should
+  be required.
+- simplifying the core's CI builds: the build should be self-contained using GitHub services which are docker
+  containers. Checking for the presence of account keys or other service credentials in CI builds is thus obsolete.
+- reducing build time: not having to build and test many extensions everytime will significantly improve the developer
+  experience, because build times will be shorter.
+- decoupling responsibilities: contributing in one repository requires only specific knowledge about that
+  particular tech, as opposed to: understanding of the _entire_ code base. Maintainer teams with specific knowledge can
+  be established.
+- different contribution criteria: while the criteria for adoption in the core should be kept very strict, the same
+  might
+  not necessarily be true for technology repos. There, features could get adopted simply for the sake of having them, as
+  long as they are properly maintained. While in the core there would have to be a specific reason to _adopt_ a feature,
+  in the technology repos there would have to be a specific reason to _deny_ them.
+- decoupling of lifecycles: archiving/abandoning a particular technology repo would not influence the core at all,
+  should we ever need to do that.
+
+## Approach
+
+At the time of this writing three technology repositories can be identified:
+
+- `technology-azure`
+- `technology-aws`
+- `technology-gcp`
```

Comment:
I would keep the current convention for repository naming:
- `Technology-Azure`
- `Technology-AWS`
- `Technologi-GCP`

not sure about the dashes tho